ssl/record/rec_layer_s3.c

```diff
@@ -572,6 +572,7 @@ int ssl_release_record(SSL_CONNECTION *s, TLS_RECORD *rr, size_t length)
         /* We allocated the buffers for this record (only happens with DTLS) */
```

Comment:
Still hoping you might add a comment either here, or above the function to document the semantics of `length == 0`.